freedomofpress/securedrop-client
# Use securedrop sdk 0.3.0

# Description

Follow up for https://github.com/freedomofpress/securedrop-sdk/issues/160

# Test Plan

1. `rm -r .venv && make venv && source .venv/bin/activate`
2. start the client against a server (a dev server running in a docker container is fine)
3. select a source and delete files and messages for this source using the new menu option in the ... menu in the upper right-hand corner
- [ ] Confirm the operation was successful (you should see all files, messages, and replies deleted in the conversation view with a message to confirm this)
